DuggaSys/microservices/sectionedService/updateListentrie_ms.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+date_default_timezone_set("Europe/Stockholm");
+include_once ('../../../Shared/sessions.php');
+include_once "../../../Shared/basic.php";
+include_once ('../shared_microservices/getUid_ms.php');
+
+// Connect to database and start session.
+pdoConnect();
+session_start();
+
+$opt=getOP('opt');
+$motd=getOP('courseid');
+$readonly=getOP('coursevers');
+
+$coursename=getOP('coursename');
+$sectid=getOP('lid');
+$sectname=getOP('sectname');
+$comments=getOP('comments');
+$highscoremode=getOP('highscoremode');
+$feedbackenabled =getOP('feedback');
+$feedbackquestion =getOP('feedbackquestion');
+$courseid=getOP('courseid');
+$coursevers=getOP('coursevers');
+$link=getOP('link');
+$kind=getOP('kind');
+$moment=getOP('moment');
+$visibility=getOP('visibility');
+$grptype=getOP('grptype');
+$tabs=getOP('tabs');
+$gradesys=getOP('gradesys');
+
+if($feedbackenabled != 1){
+	$feedbackenabled = 0;
+}
+
+if(strcmp($opt,"UPDATE")===0) {
```

Comment:
I think this (row 37)might not be needed, if the file is executing i would imagine the opt has already been compared.